src/design/Filepicker/index.stories.tsx

```diff
@@ -0,0 +1,105 @@
+import { File, Folder, FolderUp } from "lucide-react";
+
+import {
+  Filepicker,
+  FilepickerHeading,
+  FilepickerList,
+  FilepickerListItem,
+  FilepickerSeparator,
+} from "./";
+
+import type { Meta, StoryObj } from "@storybook/react";
+
+const meta = {
+  title: "Components/Filepicker",
+  component: Filepicker,
+} satisfies Meta<typeof Filepicker>;
+
+export default meta;
+type Story = StoryObj<typeof meta>;
+
+export const Default: Story = {
+  render: ({ ...args }) => (
+    <Filepicker {...args}>content goes here...</Filepicker>
+  ),
+  argTypes: {},
+};
+
+export const WithFewItems = () => (
+  <Filepicker>
+    <FilepickerHeading>Collection of Files</FilepickerHeading>
+    <FilepickerSeparator />
+    <div className="w-full hover:bg-gray-3 dark:hover:bg-gray-dark-3">
```

Comment:
These wrapper divs can be removed, right? In my understanding the `FilepickerListItem` is already rendering that div